ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -688,6 +692,9 @@ def _serialize_workflow_trigger(self) -> Optional[JsonArray]:
                     exec_config = self._serialize_integration_trigger_exec_config(trigger_class)
                     trigger_data["exec_config"] = exec_config
 
+                    # Validate trigger attributes against the expected types from the API
+                    self._validate_integration_trigger_attributes(trigger_class, trigger_attributes)
```

Comment:
[oos] lately I've been feeling that we have a ton of trigger specific logic in `base_workflow_display` that would be nice to have in a dedicated base_trigger_display. then when we delete all the `_display` classes, it will move to the referenced classes directly